lib/tpm-support-lib.py

```diff
@@ -0,0 +1,65 @@
+import re
+
+from robot.api.deco import keyword
+
+
+@keyword("Parse TCG2 Menu Snapshot Into Construction")
+def parse_tcg2_menu_snapshot_into_construction(
+    menu: str, remove_lines=None, merge_empty_options=True
+):
+    """Breaks grabbed menu data into list of (option, value) pairs.
+
+    Correctly merges multiline names and values if values end in comma
```

Comment:
@m-iwanicki I am not sure if I understand this description, this part about comma is only for `values` part of menu?

Reply:
If value ends with comma that means that next line will be continuation of this line, i.e.

| | |
|-|-|
| Bios Supported Hash | SHA1, SHA256, SHA384, |
|Algorithm | SHA512 |

Thanks to that comma it's possible to merge those 2 lines into one part.